gapis/service/service.proto

```diff
@@ -442,8 +442,12 @@ message ResourcesByType {
 message Resource {
   // The resource instance unique identifier.
   path.ID id = 1;
-  // The resource name.
-  string name = 2;
+  // The resource identifier used for display.
+  string handle = 2;
+  // The resource label.
+  string label = 4;
```

Comment:
Superdooperubernit: While we're still pre-stable, I'm happy to break protos for the sake of a simple ordering.